plugins/login-plugin/src/Components/UIComponents/Storefront/StoreFrontTv/EventDateTitle.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import Label from "../../Label";
+import { mapKeyToStyle } from "../../../../Utils/Customization";
+import PropTypes from "prop-types";
+
+const EventDateTitle = (props) => {
+  const {
+    screenStyles,
+    payload: { extensions = {} },
+  } = props;
+
+  const timestamp = extensions?.event_date;
+  if (!timestamp) {
+    return null;
+  }
+
+  const fontStyles = mapKeyToStyle("event_schedule_text", screenStyles);
+
+  const styles = StyleSheet.create({
+    container: {
+      marginTop: 30,
+    },
+    text: {
+      ...fontStyles,
+      alignSelf: "center",
+      textAlign: "center",
+    },
+  });
```

Comment:
We can memoise it to avoid re-creating styles object on every render. 
```suggestion
  const fontStyles = React.useMemo(() => mapKeyToStyle("event_schedule_text", screenStyles), [screenStyles]);

  const styles = React.useMemo(() => StyleSheet.create({
    container: {
      marginTop: 30,
    },
    text: {
      ...fontStyles,
      alignSelf: "center",
      textAlign: "center",
    },
  }), [fontStyles]);
```